src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/05.md

```diff
@@ -1,75 +1,75 @@
+However, we might still get a real medieval *gruitbeer*. One of the most popular beer varieties from those days called ‘mumme’ (aka *mum* or *mumm*), which emerged in the 14th century, was so popular that it persisted almost unchanged until the 17th century. The recipe was written down several times, and comprised wheat malt with an addition of oats and beans, fir and birch tree-tops, elderberry, cardamom seeds, bay leaf, a lot of herbs — thistle, dewdrop, burnet, betony, marjoram, gravilat, marsh mint, thyme, — and fresh eggs[ref:hornsey-history:388-389]().
 
-The beer enthusiasts had reconstructed *mumme* based on those recipes, and the style now enjoys some demand. Known examples are:
+The beer enthusiasts had reconstructed *mumme* based on those recipes, and the style now enjoys some segmented demand. Known examples are:
 
- * Mumm by Scratch Brewing Company;
+ * Mumm by Scratch Brewing Company
 
- * Hansa Mumme brewed in collaboration by 7 Fjell and Vaat Alte;
+ * Hansa Mumme brewed in collaboration by 7 Fjell and Vaat Alte
 
- * Schiøtz Mørk Mumme by Albani Bryggerierne;
+ * Schiøtz Mørk Mumme by Albani Bryggerierne
 
- * Kongens Bryghus Julemumme by Husbryggeriet Jacobsen.
+ * Kongens Bryghus Julemumme by Husbryggeriet Jacobsen
 
-Just don't get it mixed up: the real *mumme* must be dark, thick, alcoholic, and possess quite a peculiar taste closer to a coke than beer. Thin light beer proudly produced in Braunschweig under this name (as well as an energy drink and a sweet paste) is directly related to the former glory of the Braunschweig Mumme, but totally lost any resemblance to the original recipe over all those years.
+But let's not get it mixed up: A real *mumme* must be dark, thick, alcoholic, and possess quite a peculiar taste, closer to a Coke than beer. Thin light beer proudly produced in Braunschweig under this name (as well as an energy drink and a sweet paste) is directly related to the former glory of the Braunschweig Mumme, but has totally lost any resemblance to the original recipe over the years.
 
-#### Etymological
+#### Etymological Considerations
 
-All this confusion with the word *gruit* switching its meaning from ‘starter’ or ‘pastry’ to ‘a set of herbs for beer-making’ is quite a characteristical one. Almost every term related to brewing lost its original meaning, sometimes changing to quite the opposite one.
+All this confusion with the word *gruit* switching its meaning from ‘starter’ or ‘pastry’ to ‘a set of herbs for beer-making’ is quite an enigmatic one. Almost every term related to brewing lost its original meaning, sometimes changing to quite the opposite one.
```

Comment:
The intent was rather 'typical'

Reply:
Yes, I got you. Typical in the occurrence or practical application, I thought quite enigmatic to historians as its so hard to pin down. I'll follow you on this one.